components/airdrop/AirdropNft.vue

```diff
@@ -0,0 +1,464 @@
+
+        <div class="mb-6">
+          <div class="flex justify-between items-center mb-2">
+            <div class="font-semibold">
+              {{ $t('airdrop.recipientAddresses') }}
+            </div>
+            <div
+              :class="addressCounterClass"
+              class="text-sm px-2 rounded-full"
+            >
+              <div v-if="distributionMode === DistributionMode.ONE_PER_ADDRESS">
+                {{ $t('airdrop.max', [validAddressCount, totalNftCount]) }}
+              </div>
+              <div v-if="distributionMode === DistributionMode.RANDOM">
+                {{ validAddressCount }} {{ $t('airdrop.addresses') }}
+              </div>
+            </div>
+          </div>
+          <div class="text-sm opacity-50 mb-5">
+            {{ $t('airdrop.enterOneWalletAddressPerLine') }}
+          </div>
+          <textarea
+            v-model="batchAddressesInput"
+            class="bg-background-color w-full min-h-[120px] p-3 border border-gray-200 text-text-color resize-none focus:outline-none"
+            :placeholder="`${KODADOT_DAO}\n${KODADOT_DAO}`"
+            @input="handleBatchAddressesInput"
+          />
+
+          <div
+            v-if="invalidAddressList.length"
+            class="mb-4"
+          >
+            <div class="flex items-center justify-between my-4">
+              <div class="font-medium">
+                {{ $t('airdrop.addressValidation') }}
+              </div>
+              <div class="flex items-center gap-2">
+                <KIcon
+                  name="i-mdi:alert-outline"
+                  size="small"
+                  class="text-k-red"
+                />
+                {{ invalidAddressList.length }} {{ $t(invalidAddressList.length > 1 ? 'airdrop.errors' : 'airdrop.error') }}
+                <NeoButton
+                  variant="outlined-rounded"
+                  class="!w-[30px] !h-[30px] !min-w-[unset] p-0"
+                  :icon="invalidAddressExpanded ? 'chevron-up' : 'chevron-down'"
+                  size="small"
+                  @click="invalidAddressExpanded = !invalidAddressExpanded"
+                />
+              </div>
+            </div>
+
+            <div
+              v-if="invalidAddressExpanded"
+              class="space-y-2"
+            >
+              <div
+                v-for="({ index, invalidReason }, idx) in invalidAddressList"
+                :key="idx"
+                class="flex items-center gap-4 p-2 bg-k-red-accent-2 text-sm"
+              >
+                <KIcon
+                  name="i-mdi:alert-circle-outline"
+                  class="text-k-red"
+                />
+
+                <div>
+                  {{ $t('airdrop.line') }} {{ index + 1 }}: {{ invalidReason }}
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <AddressFormatWarning
+            v-if="addressPairNeedToBeFixed.length"
+            @change="handleAddressAutoCorrection"
```

Comment:
if the input is empty no errors should be shown

how to reproduce:
type something
remove 
error it's still shown

<img src=https://github.com/user-attachments/assets/455888ac-2acc-49bc-85cc-5b8bbf38de47 width=400 />